.github/workflows/ci-release.yml

```diff
@@ -31,13 +31,16 @@ jobs:
   lint:
     uses: ./.github/workflows/lint.yml
 
+  markdown-linter:
+    uses: ./.github/workflows/markdown-linter.yml
```

Comment:
do we want to use out fork of this so that we can ignore certain links? other wise I think we could end up in a situation where main and the release branch pass but this fails